.github/workflows/label-releases-doc-prs.yml

```diff
@@ -2,14 +2,17 @@
 name: "Label doc/releases PRs so our backport GitHub Action picks them up"
 
 on:
-  pull_request:
+  pull_request_target:
     branches:
       - main
     paths:
       - 'doc/releases/**'
 
 jobs:
   if_merged:
+    permissions:
+      contents: read
+      pull-requests: write
```

Comment:
> Happy to explain in detail [...]

A detailed explanation in the commit message rather than just a link always helps.  What is issue and how does the change address it?  Reviewers shouldn't have to reverse engineer this...

After perusing Github docs, I suspect that the issue is that `pull_request` trigger does not give write permission to the target repository and therefore workflows triggered from branches in external forks (as opposed to workflows triggered from branches in ceph/ceph, which get special treatment) can't assign the backport label?

Reply:
Correct

Reply:
Please update the commit message with that.